meerk40t/core/spoolers.py

```diff
@@ -329,6 +329,25 @@ def __init__(self, label, items, driver=None, priority=0, loops=1):
 
         self._stopped = True
 
+        self._estimate = 0
+        MILS_IN_MM = 39.3701
+
+        for item in self.items:
+            time_cuts = 0
+            time_travel = 0
+            time_extra = 0
+            if isinstance(item, CutCode):
+                travel = item.length_travel()
+                cuts = item.length_cut()
+                travel /= MILS_IN_MM
+                cuts /= MILS_IN_MM
```

Comment:
Eventually this needs to do better jobs based on the driver more. Since there's different native units between different values. This will work for M2Nano, Moshi, and GRBL (the grbl boards use MM and IN but really is written to use Mils since those other units are fractional)

Reply:
Acknowledged, the overall estimate algorithms need to be improved